### PR TITLE
Update nl.m3u

### PR DESCRIPTION
**Pull request by Carlinhos027**
Added new channels and missed IDs

### CODE REVIEW

**Review by UltraHDR — DISMISSED** [review]

File: streams/nl.m3u [hunk 1]
```diff
@@ -127,39 +127,119 @@ https://video.uitzending.tv:19360/zo34/zo34.m3u8
...
+#EXTINF:-1 tvg-id="HTRTV.nl",HTR TV (720p)
+https://ms7.mx-cd.net/tv/286-3043214/HTR_Media_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="LangstraatTV.nl",Langstraat TV (720p)
+https://d1qpgfi3ivud4k.cloudfront.net/nlpo/clr-nlpo/langstraat/index.m3u8
+#EXTINF:-1 tvg-id="LEOMiddelseTV.nl",LEO Middelsé TV (480p)
+https://ms7.mx-cd.net/tv/238-2061506/Omroep_LEO_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="LINQTV.nl",LINQ TV (480p)
+https://ms7.mx-cd.net/dtv-09/246-2090022/Linq_Media_TV.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="AssenTV.nl",Assen TV (720p)
+https://n1.cdn.jetstre.am/session/a179f3e35120dedc2c1ce303188a0be4/sz/OmroepAssen/webdav/live/OmroepAssenTV/playlist.m3u8?token=NwN4jakl9bcUTrZdSIVRVQ&time=1675605989
```
Comment (LaneSh4d0w) [inline streams/nl.m3u:197]: This one is tokened.

Comment (LaneSh4d0w) [reply streams/nl.m3u:197]: though I have another one : RTV Drenthe https://cdn.rtvdrenthe.nl/live/rtvdrenthe/tv/index.m3u8

Comment (LaneSh4d0w) [reply streams/nl.m3u:197]: Radio is https://rrr.sz.xlcdn.com/?account=OmroepAssen&file=RadioOmroepAssen&type=live&service=icecast&protocol=https&port=8000&output=pls
TV is https://takeoff.jetstre.am/?account=OmroepAssen&file=OmroepAssenTV%2Fplaylist.m3u8&type=live&service=webdav&protocol=https&output=playlist.m3u8

File: streams/nl.m3u [hunk 2]
```diff
@@ -127,39 +127,119 @@ https://video.uitzending.tv:19360/zo34/zo34.m3u8
...
+#EXTINF:-1 tvg-id="RTVVeluwezoomTV.nl",RTV Veluwezoom (720p)
+https://dt4pogjgi3q8s.cloudfront.net/nlpo/clr-nlpo/rtvveluwezoom/index.m3u8
+#EXTINF:-1 tvg-id="SlotstadTV.nl",Slotstad TV (720p)
+https://d3eq9mufmyqgh4.cloudfront.net/nlpo/clr-nlpo/slotstadtv/index.m3u8
+#EXTINF:-1 tvg-id="BredaNuTV.nl",BredaNu TV (720p)
+https://rtmp.bredanu.nl/live/tv.m3u8
+#EXTINF:-1 tvg-id="GOTV.nl",GO-TV (720p)
+https://593aed234297b.streamlock.net/gotvsjoerd/gotvsjoerd/playlist.m3u8
+#EXTINF:-1 tvg-id="NH.nl",NH Niews (1080p)
+https://rrr.sz.xlcdn.com/?account=nhnieuws&file=nhradio&type=live&service=wowza&protocol=https&output=playlist.m3u8
```
Comment (LaneSh4d0w) [inline streams/nl.m3u:243]: NH Radio Nieuws

TV is https://takeoff.jetstre.am/?account=nhnieuws&file=live&type=live&service=wowza&protocol=https&output=playlist.m3u8

**Review by BellezaEmporium — CHANGES_REQUESTED** [review]
See comments.

File: streams/nl.m3u (hunk 1 above)
Comment (Carlinhos027) [reply streams/nl.m3u:197]: > This one is tokened.

Got the links from a netherlands forum. Link still work since 1 year ago even with token so idk lol

Comment (LaneSh4d0w) [reply streams/nl.m3u:197]: Rather odd then, doesn't seem to work on my side.

**Review by BellezaEmporium — DISMISSED** [review]

**Review by BellezaEmporium — APPROVED** [review]

**Review by PopeyeTheSai10r — APPROVED** [review]

**Review by UltraHDR — APPROVED** [review]

**Pull request merged**